src/components/select/select.scss

```diff
@@ -79,6 +79,7 @@
                 100% - #{functions.pxToRem(16)}
             ); //This forces the label to truncate when container is too little.
             &.mdc-floating-label--float-above {
+                font-size: shared_input-select-picker.$cropped-label-hack--font-size;
```

Comment:
I'm not sure if the font just happened to be the correct size already, and this was just to avoid a potential future bug, if we or MDC changes the font size, or if I'm not looking in the right place, but I'm not seeing any difference in the floating label size for `limel-select` in this branch vs `next`? 🤔

Reply:
Yes, the font-size should not be touched in this PR, seems to be done by mistake